zeppelin-web/src/app/visualization/builtins/visualization-scatterchart.js

```diff
@@ -70,16 +70,15 @@ export default class ScatterchartVisualization extends Nvd3ChartVisualization {
     chart.yAxis.tickFormat(function(d) {return self.yAxisTickFormat(d, self.yLabels);});
 
     // configure how the tooltip looks.
-    chart.tooltipContent(function(key, x, y, graph, data) {
-      var tooltipContent = '<h3>' + key + '</h3>';
-      if (self.config.size &&
-        self.isValidSizeOption(self.config, self.tableData.rows)) {
-        tooltipContent += '<p>' + data.point.size + '</p>';
-      }
-
+  /*  chart.tooltip.contentGenerator(function(d) {
```

Comment:
I'd just remove it instead of commenting it out.
Similarly here, do you have a screenshot?